dojo-starter/src/systems/actions.cairo

```diff
@@ -3,7 +3,8 @@ use dojo_starter::models::game::Game;
 // define the interface
 #[dojo::interface]
 trait IActions {
-    fn start_game() -> Game;
+    fn start_game(world:IWorldDispatcher) -> Game;
+    fn end_game(game:Game) -> None;
```

Comment:
ideally you will get an id from the frontend

Reply:
we need to have some sort of collection in the code. could be a vector or a dictionary/map. how to fetch the game object using the game Id otherwise?

Reply:
we already have that, you can call `get!(world, game_id, (GAME))` and you will get a game. for reference see here: https://github.com/Mugen-Space/dojo-game-jam-v1/blob/main/dojo-starter/src/store.cairo#L17 and here is how you can use it: https://github.com/Mugen-Space/dojo-game-jam-v1/blob/main/dojo-starter/src/systems/actions.cairo#L123

Reply:
okay